Engine/AppManagerPrivate.cpp

```diff
@@ -279,6 +279,12 @@ AppManagerPrivate::AppManagerPrivate()
     setMaxCacheFiles();
 
     runningThreadsCount = 0;
+
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0) && defined(__NATRON_LINUX__)
```

Comment:
This reminds me that we should use standard preprocessor symbols everywhere rather than `__NATRON_*__`. Let's do it later.

Reply:
I was thinking of using the `Q_OS_*` definitions, they provide a good detection for other Unix and BSD systems too.